docs/other/troubleshooting-gratia.md

```diff
@@ -0,0 +1,375 @@
+    2014-06-03 10:00:49 CDT Gratia: condor_meter --history: Usage records submitted: 399
+    2014-06-03 10:00:49 CDT Gratia: condor_meter --history: Usage records found: 400
+    2014-06-03 10:00:49 CDT Gratia: RUNNING condor_meter MANUALLY Finished
+
+    %BLUE% or if you need to go back to the beginning of time%ENDCOLOR%
+    root@host # /usr/share/gratia/condor/condor_meter --history --verbose
+    2014-06-03 10:06:19 CDT Gratia: RUNNING condor_meter MANUALLY using all HTCondor history
+    2014-06-03 10:06:19 CDT Gratia: RUNNING: condor_history -l
+    2014-06-03 10:11:38 CDT Gratia: condor_meter --history: Usage records submitted: 13026
+    2014-06-03 10:11:38 CDT Gratia: condor_meter --history: Usage records found: 13027
+    2014-06-03 10:11:38 CDT Gratia: RUNNING condor_meter MANUALLY Finished
+
+
+Not much is printed to the screen, but you can see progress in the Gratia log file:
+
+    13:35:28 CDT Gratia: Initializing Gratia with /etc/gratia/condor/ProbeConfig
+    13:35:28 CDT Gratia: Creating a ProbeDetails record 2012-04-04T18:35:28Z
+    13:35:28 CDT Gratia: ***********************************************************
+    13:35:28 CDT Gratia: OK - Handshake added to bundle (1/100)
+    13:35:28 CDT Gratia: ***********************************************************
+    13:35:28 CDT Gratia: List of backup directories: [u'/var/lib/gratia/tmp']
+    13:35:28 CDT Gratia: Reprocessing response: OK - Reprocessing 0 record(s) uploaded, 0 bundled, 0 failed
+    13:35:28 CDT Gratia: After reprocessing: 0 in outbox 0 in staged outbox 0 tar files
+    13:35:28 CDT Gratia: Creating a UsageRecord 2012-04-04T18:35:28Z
+    ...
+    13:35:29 CDT Gratia: Processing bundle file: 
+    13:35:29 CDT Gratia: Processing bundle file: /var/lib/gratia/tmp/gratiafiles/
+        subdir.condor_fermicloud084.fnal.gov_gratia-osg-itb.opensciencegrid.org_80/
+        outbox/r.18425.condor_fermicloud084.fnal.gov_gratia-osg-itb.opensciencegrid.org_80.gratia.xml__BSuXo18428
+    ...
+    13:35:29 CDT Gratia: ***********************************************************
+    13:35:29 CDT Gratia: Removing log files older than 31 days from /var/log/gratia
+    13:35:29 CDT Gratia: /var/log/gratia uses 0.035% and there is 73% free
+    13:35:29 CDT Gratia: Removing incomplete data files older than 31 days from /var/lib/gratia/data/
+    13:35:29 CDT Gratia: /var/lib/gratia/data uses 0% and there is 73% free
+    13:35:29 CDT Gratia: End of execution summary: new records sent successfully: 37
+
+
+
+!!! note 
+    Condor rotates history files, so you can only report what Condor has kept. Controlling the Condor history is documented in the Condor manual. In particular, see the options for [MAX\_HISTORY\_LOG](http://research.cs.wisc.edu/condor/manual/v7.6/3_3Configuration.html#param:MaxHistoryLog) and [MAX\_HISTORY\_ROTATIONS](http://research.cs.wisc.edu/condor/manual/v7.6/3_3Configuration.html#param:MaxHistoryRotations).
```

Comment:
Let's update the link to a modern version of condor

Reply:
Is there a way to link directly to a parameters?

Reply:
I normally search for the variable in the index and find the relevant link for the configuration section (3.5 for 8.6). Here are the links:

- http://research.cs.wisc.edu/htcondor/manual/v8.6/3_5Configuration_Macros.html#21780
- http://research.cs.wisc.edu/htcondor/manual/v8.6/3_5Configuration_Macros.html#21785

Reply:
Do you crack open the raw html to get these numbers?  Do these numbers change for future releases?

Reply:
Nope, I'm not that sophisticated. Here's my process:

1. Go [here](http://research.cs.wisc.edu/htcondor/manual/v8.6/Index.html)
1. Search for the variable in question
1. Right-click and copy the link location for the 3.5.1 link

I believe the numbers do change between releases. @timtheisen may have details on how that works.

Reply:
Ah, ok.  That index, didn't get that.  Makes sense, Thanks.